Mathlib/Analysis/Normed/Ring/Seminorm.lean

```diff
@@ -151,6 +151,26 @@ end Ring
 
 end RingSeminorm
 
+/-- If `f` is a ring seminorm on `a`, then `∀ {n : ℕ}, n ≠ 0 → f (a ^ n) ≤ f a ^ n`. -/
+theorem map_pow_le_pow {F α : Type _} [Ring α] [FunLike F α ℝ] [RingSeminormClass F α ℝ] (f : F)
```

Comment:
It's better to use `Type*` as opposed to `Type _` , and not just here but most other places in this PR.